save.go

```diff
@@ -288,10 +302,35 @@ func copySrc(dir string, deps []Dependency) error {
 				ok = false
 			}
 		}
+
+		// check parent directorys for legal files and copy
+		for _, parent := range parents(rel) {
```

Comment:
There's no need to try to find the parent, vf already contains all files for the repo that contains the package. I realized that I didn't update the listFIles() doc comment. So I did: 2115b735b60387f29297a05fcfed85c77e40530f. Was that a source of confusion?